src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -653,7 +653,8 @@ private static void doExec(PrintStream in, PrintStream out, boolean[] masks, Str
 
         // BourneShellScript.launchWithCookie escapes $ as $$, we convert it to \$
         for (String cmd : starter.cmds()) {
-            String fixedCommand = cmd.replaceAll("\\$\\$", "\\\\\\$");
+            String fixedCommand = cmd.replaceAll("\\$\\$", "\\\\\\$")
+                    .replaceAll("\\\"", "\\\\\"");
```

Comment:
:groan:

@kerogers-cloudbees you were also looking at this code IIRC?

Reply:
I was looking at this, but I ended up not doing anything with it yet because this where the commands are edited, not the env vars. 

FWIW I do think using `Matcher.quoteReplacement("\\$")` is easier to read than `"\\\\\\$`